src/BlazorWebView/samples/MauiRazorClassLibrarySample/MauiRazorClassLibrarySample.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFrameworks>$(_MauiDotNetTfm);$(_MauiPreviousDotNetTfm)</TargetFrameworks>
+    <TargetFrameworks>$(_MauiDotNetTfm)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(IncludePreviousTfms)' != 'false'">$(TargetFrameworks);$(_MauiPreviousDotNetTfm)</TargetFrameworks>
```

Comment:
I think we should invert the logic here and make it `'$(IncludePreviousTfms)' == 'true'`